h/models/es_sync_task.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, Integer, text
+
+from h.db import Base
+from h.db.types import URLSafeUUID
+
+
+class EsSyncTask(Base):
```

Comment:
The table is a queue, but each individual row in the table is a "task" (or a "message"?). Similar to for example `models.Annotation` rather than `Annotations`